### ISSUE TITLE
[Request] Asset delete button

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#432]
I see a bunch of tokens appearing that I do not want to be associated with. 

Therefore, I would like to request a button next to assets that makes it easy to send the asset to a burn address. I can foresee spammers sending offensive/adds/scam tokens to addresses repeatedly and very easily. With the upcoming messaging system, this will only become worse.

Make it go for example to RXBurnXXXXXXXXXXXXXXXXXXXXXXWUo9FV

**Comment by TronBlack** [comment]
Yes, we agree.  Will likely hide any assets that go to a previously used address.
And easily burn (with confirmation).

**Comment by jeroz1** [comment]
Thanks for the followup and happy to see others liking the idea :). 

What do you mean with 'Will likely hide any assets that go to a previously used address.'?

**Comment by wolfsokta** [comment]
Spam tokens that you don't want are likely going to be sent to an address that already contains RVN or another asset. If that is the case that token could be moved to a hidden/spam tab.

**Comment by TronBlack** [comment]
This will be addressed in the GUI as part of the anti-spam effort.

**Comment by dregimbal** [comment]
I wouldn't want to burn these spam tokens (due to the transaction fee), but an optional "hidden" tab would be good

**Comment by rmarjama** [comment]
I was hoping this obvious problem would be addressed.  One more aspect of this, for me, is very important in any fix being done.  I do not use any common passwords, passphrases where things of value are at risk.  I do not store this passwords digitally.  They are manually written and kept in safe storage when they are not being used.  Obviously, my wallet passphrase is one of these types of passphrases.  Also, since I am a overly cautious person (no comments please), I would really like it if this method of deleting (or whatever will be used), could be completed where I do not have to expose my passphrase at all.  Obviously because of the inconvenience, but more important, I am uncertain to any safeguards that may be in place that would protect my passphrase from being compromised by the asset itself, if it somehow were coded to capture the passphrase and send it to a specific address on the blockchain when any transfer of the asset occurs.  Obviously I do not know if this is possible, but if their is any possibility at all, or some other way my passphrase could be compromised, no matter how improbable, giving a passphrase when working with a useless asset seems more undone, then it should be.  Thanks for reading this long request.  And thanks for all the work being done on Ravencoin.  I am a firm believer in where this coin is going and how much real use it will be to so many everywhere. I have 3 rigs that have been mining it for over 8 months now full time.

**Comment by jeroz1** [comment]
So the idea of the implementation is to not actually delete the spam (by sending it to a burn address) but to hide it so that you don't see it anymore. Deleting it would indeed burden you with signing transactions but also miner fees. @rmarjama

**Comment by rmarjama** [comment]
> So the idea of the implementation is to not actually delete the spam (by sending it to a burn address) but to hide it so that you don't see it anymore. Deleting it would indeed burden you with signing transactions but also miner fees. @rmarjama

Thanks for the reply.  As long as there can be no activity from the token by some means, that would be better than having it visible.  I would prefer to have a simple delete button that sends it to be burned without exposing my passphrase to do it however.  The cost is minimal enough, so I don't think that is a major item.  Besides, there should be a way to build this into the asset layer coding so burning unwanted tokens reverts any fees back to the issuer of the token.  Maybe double charge a transfer fee to begin with if necessary and the burn can be without fee then.

**Comment by cfox** [comment]
Hiding assets is on the roadmap...

**Comment by jeroz1** [comment]
requesting to open this again to get back priority on hiding unwanted assets

**Comment by HyperPeek** [comment]
Agree, reopen
